.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip
```

Comment:
Would be good to make this a separate PR and use the latest version, 3.6.3 currently

Reply:
@salmaanrizvi  
I agree with @timja here.  I'd rather the discuss the pros/cons of adding maven wrapper in another PR.